packages/react-native-bpk-component-calendar/src/ios/CalendarBridge/RCTBPKCalendarManager.m

```diff
@@ -45,6 +50,40 @@ - (UIView *)view {
 
 RCT_EXPORT_VIEW_PROPERTY(onDateSelection, RCTBubblingEventBlock)
 
+
+/*
+ * When the calendar renders in certain configurations the initial
+ * render is incorrect. With this method, called from `componentDidMount`,
+ * is called the calendar is forced to re-render to fix the bug.
+ */
+RCT_EXPORT_METHOD(forceRender:(nonnull NSNumber *)reactTag) {
+    [self.bridge.uiManager addUIBlock:
+     ^(__unused RCTUIManager *uiManager, NSDictionary<NSNumber *, UIView *> *viewRegistry){
+         UIView *view = viewRegistry[reactTag];
+
+         if ([view isKindOfClass:[BPKCalendar class]]) {
+             BPKCalendar *calendar = (BPKCalendar *)view;
+             NSArray<NSDate* > *selectedDates = calendar.selectedDates;
+
+             calendar.selectedDates = @[];
```

Comment:
Do we need to empty the calendar once for it to render correctly on the second reload? Is this a temporary fix or is this going to be around?

Reply:
It seems to be required yeah